diff --git a/node_modules/react-native-collapsible/index.d.ts b/node_modules/react-native-collapsible/index.d.ts
index 26f8d56..beea9bd 100644
--- a/node_modules/react-native-collapsible/index.d.ts
+++ b/node_modules/react-native-collapsible/index.d.ts
@@ -95,4 +95,5 @@ export interface CollapsibleProps {
   onAnimationEnd?: () => void;
 }
 
-export default class Collapsible extends React.Component<CollapsibleProps> {}
+// https://github.com/oblador/react-native-collapsible/issues/450, https://github.com/oblador/react-native-collapsible/issues/436
+export default class Collapsible extends React.Component<React.PropsWithChildren<CollapsibleProps>> {}
